gnark-prover/main.go

```diff
@@ -581,6 +539,13 @@ func LoadKeysFromConfigOrInline(context *cli.Context) ([]*prover.ProvingSystem,
 	return pss, nil
 }
 
+func getKeysByArgs(context *cli.Context) []string {
+	var circuitDir = context.String("circuit-dir")
+	var inclusion = context.Bool("inclusion")
+	var nonInclusion = context.Bool("non-inclusion")
+	return prover.GetKeys(circuitDir, inclusion, nonInclusion)
```

Comment:
does this enable a server to be instantiated with both inclusion, combined and non-inclusion proving keys at the same time?

Reply:
combined is missing right?